recipes/cuda-cccl-impl/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/lib/cmake
+mkdir -p ${PREFIX}/include
+
+mv thrust/thrust/cmake ${PREFIX}/lib/cmake/thrust
+mv cub/cub/cmake ${PREFIX}/lib/cmake/cub
+cp -r libcudacxx/lib/cmake/libcudacxx ${PREFIX}/lib/cmake
```

Comment:
Why do we `mv` some and `cp -r` others? I think we should `cp -r` all of them unless I'm missing something. And to align with other libraries' `build.sh` scripts, let's use `-v` for verbosity in the logs.
```suggestion
cp -rv thrust/thrust/cmake ${PREFIX}/lib/cmake/thrust
cp -rv cub/cub/cmake ${PREFIX}/lib/cmake/cub
cp -rv libcudacxx/lib/cmake/libcudacxx ${PREFIX}/lib/cmake
```

Reply:
In this case the `cmake` directories get `mv`'d to the lib directory and the `thrust`/`cub` directories are then copied to the `include` directory below. So if we change this to `cp` we also need to add `rm` to maintain the expected layout.
```suggestion
cp -rv thrust/thrust/cmake ${PREFIX}/lib/cmake/thrust
rm -rv thrust/thrust/cmake ${PREFIX}/lib/cmake/thrust
cp -rv cub/cub/cmake ${PREFIX}/lib/cmake/cub
rm -rv cub/cub/cmake ${PREFIX}/lib/cmake/cub
cp -rv libcudacxx/lib/cmake/libcudacxx ${PREFIX}/lib/cmake
```

Reply:
Thanks for explaining, I missed that. This suggestion can be resolved with no changes needed.

Reply:
Updated my suggestion so we can still add `-v` and commit this, thanks!